Makefile

```diff
@@ -13,7 +13,8 @@ pelican_push:
 	docker push $(REGISTRY)/${RL_TEAM_ID}:pelican_latest
 
 pelican_test:
-	docker run $(REGISTRY)/${RL_TEAM_ID}:pelican_latest Combatant/tests/test_pelican.sh
+	PATH_TO_AGENT="data/agents/models/panther/move_north_nn"
```

Comment:
Does this line do anything, or is it an example of what the "<<PATH TO AGENT>>" should look like?

Reply:
Good catch, will remove this.